branch_tags/models.py

```diff
@@ -0,0 +1,25 @@
+from django.db                 import models
+from django.db.models.deletion import CASCADE
+
+from users.models              import User
+from postings.models           import Posting
+from keywords.models           import Keyword
+
+class User_tag(models.Model):
+    name         = models.CharField(max_length=45, unique=True)
+    user         = models.ManyToManyField(User, related_name='users')
+    created_at   = models.DateField(auto_now_add=True)
+    updated_at   = models.DateField(auto_now=True)
+
+    class Meta:
+        db_table = 'users_tags'
+
+class Posting_tag(models.Model):
+    name         = models.CharField(max_length=45, unique=True)
+    posting      = models.ManyToManyField(Posting, related_name='posts')
+    keyword      = models.ForeignKey(Keyword, on_delete=CASCADE)
```

Comment:
suggestion과 같이 작성해주시면 이미 models를 import 했었기 때문에 거기서 다시 CASCADE를 import 하지 않아도 됩니다!
```suggestion
    keyword      = models.ForeignKey(Keyword, on_delete=models.CASCADE)
```